app/models/pod.rb

```diff
@@ -25,7 +25,9 @@ class Pod < ActiveRecord::Base
 
   class << self
     def find_or_create_by(opts) # Rename this method to not override an AR method
-      u = URI.parse(opts.fetch(:url))
+      url = opts.fetch(:url)
```

Comment:
This is generally a webmock problem. It doesn't stub effective_url in https://github.com/bblimke/webmock/blob/master/lib/webmock/http_lib_adapters/typhoeus_hydra_adapter.rb.

So to match the behavior in the production environment we have to patch it.